packages/omg-js-util/src/transaction.js

```diff
@@ -92,6 +92,17 @@ const transaction = {
     return encoded
   },
 
+  // TODO ADD js-doc
+  decodeDeposit: function (encodedDeposit) {
+    const { outputs } = transaction.decodePaymentTransactionRaw(encodedDeposit)
+    const [{ outputGuard, amount, currency }] = outputs
+    return {
+      owner: outputGuard,
+      amount,
+      currency
```

Comment:
Could we just have one decode function for both deposit and normal tx. They are anyway the except that one does not have any inputs

Reply:
this is just another high level helper, `decodePaymentTransactionRaw` will work for both type